dojo/product/views.py

```diff
@@ -317,9 +317,8 @@ def finding_querys(request, prod):
     # risk_acceptances = Risk_Acceptance.objects.filter(engagement__in=Engagement.objects.filter(product=prod)).prefetch_related('accepted_findings')
     # filters['accepted'] = [finding for ra in risk_acceptances for finding in ra.accepted_findings.all()]
 
-    from dojo.finding.helper import ACCEPTED_FINDINGS_QUERY
```

Comment:
a while ago we started to centralize the definition of queries for "accepted", "closed", etc so I'd prefer it to keep using these centralized Q objects.

Reply:
@valentijnscholten Do you request a change here or do we want to accept the PR?

Reply:
just made the change, let's see if the tests go green